nms/app/packages/magmalte/app/views/equipment/EquipmentGatewayPools.js

```diff
@@ -0,0 +1,231 @@
+ */
+import type {GatewayPoolRecordsType} from '../../components/context/GatewayPoolsContext';
+import type {WithAlert} from '@fbcnms/ui/components/Alert/withAlert';
+
+import ActionTable from '../../components/ActionTable';
+import CardTitleRow from '../../components/layout/CardTitleRow';
+import GatewayPoolsContext from '../../components/context/GatewayPoolsContext';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import React from 'react';
+import SettingsInputAntennaIcon from '@material-ui/icons/SettingsInputAntenna';
+import withAlert from '@fbcnms/ui/components/Alert/withAlert';
+import {GatewayPoolEditDialog} from './GatewayPoolDetailConfigEdit';
+import {colors} from '../../theme/default';
+import {makeStyles} from '@material-ui/styles';
+import {useContext, useEffect, useState} from 'react';
+import {useEnqueueSnackbar} from '@fbcnms/ui/hooks/useSnackbar';
+import {useRouter} from '@fbcnms/ui/hooks';
+
+const useStyles = makeStyles(theme => ({
+  dashboardRoot: {
+    margin: theme.spacing(3),
+    flexGrow: 1,
+  },
+  topBar: {
+    backgroundColor: colors.primary.mirage,
+    padding: '20px 40px 20px 40px',
+  },
+  tabBar: {
+    backgroundColor: colors.primary.brightGray,
+    padding: '0 0 0 20px',
+  },
+  tabs: {
+    color: colors.primary.white,
+  },
+  tab: {
+    fontSize: '18px',
+    textTransform: 'none',
+  },
+  tabLabel: {
+    padding: '20px 0 20px 0',
+  },
+  tabIconLabel: {
+    verticalAlign: 'middle',
+    margin: '0 5px 3px 0',
+  },
+}));
+
+export default function GatewayPools() {
+  const classes = useStyles();
+  return (
+    <div className={classes.dashboardRoot}>
+      <Grid container justify="space-between" spacing={3}>
+        <Grid item xs={12}>
+          <GatewayPoolsTable />
+        </Grid>
+      </Grid>
+    </div>
+  );
+}
+
+type GatewayPoolRowType = {
+  name: string,
+  id: string,
+  mmeGroupId: number,
+  gatewayPrimary: Array<GatewayPoolRecordsType>,
+  gatewaySecondary: Array<GatewayPoolRecordsType>,
+};
+
+function GatewayPoolsTableRaw(props: WithAlert) {
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const ctx = useContext(GatewayPoolsContext);
+  const [open, setOpen] = useState(false);
+  const [currRow, setCurrRow] = useState<GatewayPoolRowType>({});
+  const [gwPool, setGwPool] = useState(ctx.state[currRow.id] || {});
+  const {history, match} = useRouter();
+  useEffect(() => {
+    setGwPool(ctx.state[currRow.id] || {});
+  }, [currRow.id, ctx.state]);
+  const gwIds = (gateways: Array<GatewayPoolRecordsType>) => {
+    return (
+      <>
+        {gateways.length > 0 ? (
+          gateways.map(gw => (
+            <>
+              <Link
+                variant="body2"
+                component="button"
+                onClick={() => {
+                  history.push(
+                    match.url.replace(`pools`, `gateway/${gw.gateway_id}`),
+                  );
+                }}>
+                {gw.gateway_id}
```

Comment:
If we have multiple gateways does this render each gateway in a separate line or will it all be right next to each other ?

Reply:
it will be in a separate line 
<img width="1149" alt="Capture d’écran 2021-03-23 à 20 23 08" src="https://user-images.githubusercontent.com/26038920/112205901-9e1f8600-8c15-11eb-98bc-d84c7f658f8d.png">